Exercise1.js

```diff
@@ -0,0 +1,6 @@
+function calculateAverage(numbers) {
+    let sumOfNumbers = numbers.reduce((acc, val) => acc + val,0)
+    const average = sumOfNumbers / numbers.length
+    return average
```

Comment:
No need of `average` variable, you can just return the calculation of the average